WrightTools/artists/_quick.py

```diff
@@ -286,6 +298,30 @@ def quick2D(
         plt.yticks(fontsize=14)
         ax.set_xlabel(xaxis.label, fontsize=18)
         ax.set_ylabel(yaxis.label, fontsize=18)
+        ax.grid()
+        # lims
+        ax.set_xlim(xlim)
+        ax.set_ylim(ylim)
+        # add zero lines
+        plt.axvline(0, lw=2, c="k")
+        plt.axhline(0, lw=2, c="k")
+        # add constants to title
+        ls = []
+        for constant in d.constants:
+            ls.append(constant.label)
+        title = ", ".join(ls)
+        ax.set_title(title)
+        # variable marker lines
+        for constant in d.constants:
+            if not constant.units == None:
+                # x axis
+                if xaxis.units_kind == constant.units_kind:
+                    constant.convert(xaxis.units)
+                    plt.axvline(constant.value, color="k", linewidth=4, alpha=0.25)
+                # y axis
+                if yaxis.units_kind == constant.units_kind:
+                    constant.convert(yaxis.units)
+                    plt.axhline(constant.value, color="k", linewidth=4, alpha=0.25)
         # colorbar
         cax = plt.subplot(gs[1])
```

Comment:
I think we should put `plt.sca(ax)` after plotting the colorbar so that, should users wish to modify the plot (only particularly useful for single plot calls) they can do so naturally.

That said, these plots are explicitly not for publication quality figures.
One use case I can imagine this being valuable for is to set xlim and ylim in a couple different calls to get the plot area to be the same, while still wanting the quick features like the colorbar and overlines.
A potential additional change along the same lines is to return the axis object when not autosaving (currently returns empty list)

Reply:
@ksunden I leave you to make these changes if you desire. I am indifferent.